global_segment_map_node/srv/GetAlignedInstanceBoundingBox.srv

```diff
@@ -0,0 +1,3 @@
+uint16 instance_id
```

Comment:
Would it maybe make more sense to have these service messages also in modelify_msgs?